frontend/pages/create-project.tsx

```diff
@@ -44,73 +50,117 @@ const CreateProject: React.FC = () => {
         }
     };
 
-    const addParticipant = () => {
-        if (formik.values.newList.trim() !== '') {
-            formik.setFieldValue('listOfParticipant', [...formik.values.listOfParticipant, formik.values.newList]);
-            formik.setFieldValue('newList', '');
+    const addMembers = () => {
+        if (newMembers.trim() !== '') {
+            formik.setFieldValue('members', [...formik.values.members, newMembers]);
+            setNewMember('');
         }
     };
 
-    const removeParticipant = (email: string) => {
-        const updatedParticipants = formik.values.listOfParticipant.filter(participant => participant !== email);
-        formik.setFieldValue('listOfParticipant', updatedParticipants);
+    const handleMemberChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setNewMember(e.currentTarget.value);
+    };
+
+    const removeMembers = (email: string) => {
+        const updatedMembers = formik.values.members.filter(members => members !== email);
+        formik.setFieldValue('members', updatedMembers);
     };
 
     const addMaterial = () => {
-        if (formik.values.newMaterial.trim() !== '') {
-            formik.setFieldValue('material', [...formik.values.material, formik.values.newMaterial]);
-            formik.setFieldValue('newMaterial', '');
+        if (newMaterials.trim() !== '') {
+            formik.setFieldValue('material', [...formik.values.material, newMaterials]);
+            setNewMaterials('');
         }
     };
 
+    const handleMaterialChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setNewMaterials(e.currentTarget.value);
+    };
+
     const removeMaterial = (index: number) => {
         formik.setFieldValue('material', formik.values.material.filter((_, i) => i !== index));
     };
 
+    const formik = useFormik({
+        initialValues: {
+            name: '',
+            description: '',
+            selectedFile: '',
+            members: [] as string[],
+            material: [] as string[],
+        },
+        validationSchema: validationSchema,
+        onSubmit: async (values) => {
+            try {
+                const response = await fetch(`${API_URL}/projects`, {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify(values),
+                });
+                if (!response.ok)
+                    throw new Error(`HTTP error! Status: ${response.status}`);
+                toast.success('Your project has been created!', toastConfig);
+                formik.resetForm();
+            } catch (error: unknown) {
+                toast.error(`An error occurred: ${error}`, toastConfig);
+            }
+        },
+    });
+
     return (
         <div className="flex flex-col items-center mt-24 mb-24">
             <div className="w-4/5 bg-white rounded-lg shadow-lg h-auto p-10">
```

Comment:
`w-4/5` Is this fine on every type of device ?

Reply:
for me yes